### ISSUE TITLE
Audit #1 – Download a single file – Semantics and interactivity of interstitial file screen (when downloading)

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4347]
Connected to Audit 1, see #4300 
1.4 Download a single file, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
Clickable spans, no keyboard interactivity (2.1.1), thus no focus (2.4.7), overall wrong semantics for interactive controls (1.3.1)

# Remediation
In case of an interstitial screen, e.g. .png file: Clickable spans should be buttons or links (depending on their function). Download control should be a <a> with download attribute. All controls in this construct (see screenshot) must have at least accessible names, ideally visually dicernable text (Sidenode: Click events seem broken in Firefox 82).

# Update Jan 22nd, 2021:
If the final download triggers *are* links, it should be clear from the link text that the link target is a file (WCAG: https://www.w3.org/TR/WCAG21/#link-purpose-in-context). Possible technique to remedy: Using `aria-label`: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA8 or `aria-labelledby`: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA7 Example2. Keep in mind that this is a string that possibly needs translation.

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
